doc/source/the-origin-of-cinema.md

```diff
@@ -0,0 +1,35 @@
+
+![Kinetoscope ](_static/images/the-origin-of-cinema/image2.jpg)
+
+The Kinetoscope inspired the Lumiere brothers to create their own cinematographic device based on a record where a tape rotated at 16 frames per second and managed to project the movement of people in December 1895. The success spread not only in France but throughout Europe and North America.
+
+The Lumière brothers made more than 10 films lacking natural sets and actors. Of course, the lack of editing, the haste and the fixed camera were part of the faults committed by the beginners of cinema.
+
+The first person to make a film was Alice Guy, considered the founder of narrative cinema and who came to surpass the demonstration made by the Lumiere brothers. Alice managed to lay the foundations for fiction and she was also the first person to achieve her financial support through this profession. Therefore, the first film that was generated in the history of cinema was La Fée aux Choux in 1896, which translates as "The Cabbage Fairy" and later made other films such as Sage Femme de Premiere Classe in 1902 and in 1905 made the Emerald.
+
+After the success of Guy, director Georges Meliés entered this magical profession, whose fantastic sets and fascinating stories captivated the viewer. Films such as "Bluebeard" and "Faust" created in 1901 generated new cinematographic techniques, especially with "Journey to the Moon" (1902) and also with "Journey through the impossible" made in 1904, managed to implement the technique of the theater before the cameras and made the first special effects, thus achieving science fiction films. Since then the cinema has evolved by leaps and bounds to the present day. Important directors such as Erich Von Stroheim, Murnau and Charles Chaplin emerged. Adventure films in the United States were not long in coming, such as those made by Douglas Fairbanks or the romantic dramas of Valentino. Then the American comic school that was born from the comedy made by Mack Sennet and these films were cataloged as the most beautiful and were based on stereotyping social characters such as the fat man, the policeman, the mustachioed bourgeois and the miser.
+
+We cannot forget the cinema made by Charles Chaplin and Buster Keaton, but it was not until 1927 when the first sound film called "The Jazz Singer" was released and since then, the cinema ceased to exist as it was known, a greater plot fluidity and continuity of the story, for that same year the dubbing appeared and by 1935 with the film "The vanity fair" is that the cinema appears in Technicolor, but it really reaches color perfection in 1939 with "What the wind took away
```

Comment:
```suggestion
We cannot forget the cinema made by Charles Chaplin and Buster Keaton, but it was not until 1927 when the first sound film called "The Jazz Singer" was released and since then, the cinema ceased to exist as it was known, a greater plot fluidity and continuity of the story, for that same year the dubbing appeared and by 1935 with the film "The vanity fair" is that the cinema appears in Technicolor, but it really reaches color perfection in 1939 with "What the wind took away".
```